jupyter_core/command.py

```diff
@@ -15,19 +15,21 @@
 import sys
 from subprocess import Popen
 
+from .utils.shutil_which import which
```

Comment:
We should use the stdlib implementation where it's available, like this:

https://github.com/jupyter/jupyter_core/blob/0fe5994c2ec528de4cb01178a296d95db945fa06/jupyter_core/application.py#L18-L22